util/lastfm.q

```diff
@@ -0,0 +1,146 @@
+(.lfm.console:{system"c "," "sv string 20 1000|system"c"})[];
+
+/ libs
+\l util/req.q
+\l util/log.q
```

Comment:
don't need to explicitly load these, they'll be loaded automatically by kdbslack - if concern is order, there's an `order.txt` file in `util/` that specifies files to load first (would just need to add `log.q` to that)

Reply:
NB, is a leftover from testing.